tools/esdlcmd/esdl-publish.cpp

```diff
@@ -863,7 +826,7 @@ class EsdlBindMethodCmd : public EsdlBindServiceCmd
         printf( "\n Use this command to publish ESDL Service based bindings.\n"
                 "   To bind a ESDL Service, provide the target ESP process name\n"
                 "   (esp which will host the service.) \n"
-                "   It is also necessary to provide the Port on which this service is configured to run (ESP Binding),\n"
+                "   It is also necessary to provide the target ESP binding name,\n"
```

Comment:
Does just specifying the port not work?  I think using port is much more intuitive than ESP binding name... which is usually a confusing unique id.  If it doesn't work, please open a JIRA to add it (to any command line that currently requires ESP binding name).  If it does work the help should still say so (in fact it should be considered the preferred way.